README.md

```diff
@@ -5,12 +5,12 @@ File Format, as described here:
 
 <https://aomediacodec.github.io/av1-avif/>
 
-It is a work-in-progress, but can already encode and decode all AOM supported
-YUV formats and bit depths (with alpha).
+It can encode and decode all AOM supported YUV formats and bit depths (with
```

Comment:
`AOM` refers to all Alliance for Open Media formats or to libaom? I think saying "AV1" or "AV1 main implementation libaom" with a link is clearer.

Reply:
This pull request is a cherry-pick, so I will make the change you suggested on the main branch instead.

Here AOM refers to libaom. I agree we should change AOM to AV1.